bucket/fbide.json

```
@@ -0,0 +1,30 @@
+{
+    "homepage": "https://fbide.freebasic.net",
+    "version": "0.4.6r4",
+    "license": "GPL-2.0-or-later",
+    "description": "#1 editor for FreeBASIC",
+    "url": "http://downloads.sourceforge.net/project/fbide/fbide%200.4/fbide%200.4.6%20r4/FBIde0.4.6r4.zip",
+    "hash": "sha1:326327c7082adae0af1df0281cde4af990b1e273",
+    "extract_dir": "FBIde0.4.6r4",
+    "depends": "freebasic",
+    "shortcuts": [
+        [
+            "fbide.exe",
+            "FBIde"
+        ]
+    ],
+    "pre_install": [
+        "$fbc = (versiondir 'freebasic' 'current') -replace '\\\\', '\\\\'",
+        "$pref = (Get-Content \"$dir\\IDE\\prefs_win32.ini\" -Raw) -replace 'fbc.exe', \"$fbc\\\\fbc.exe\"",
+        "$pref | Set-Content \"$dir\\IDE\\prefs_win32.ini\" -Encoding ASCII"
+    ],
+    "persist": "IDE\\prefs_win32.ini",
+    "checkver": {
+        "url": "https://fbide.freebasic.net/download",
+        "regex": ">FBIde ((?<version>[\\d.]+)(?<release>r\\d))<"
+    },
+    "autoupdate": {
+        "url": "http://downloads.sourceforge.net/project/fbide/fbide%20$majorVersion.$minorVersion/fbide%20$matchVersion%20$matchRelease/FBIde$version.zip",
```

I do not like using `$matchVersion` and `$version` in same string. It could confuse other maintainers.

```suggestion
        "url": "http://downloads.sourceforge.net/project/fbide/fbide%20$majorVersion.$minorVersion/fbide%20$matchMain%20$matchRelease/FBIde$version.zip",
```

---

Feel free to change main into something more reliable